meta-mender-core/recipes-bsp/u-boot/u-boot-mender.inc

```diff
@@ -351,6 +351,9 @@ do_save_mender_auto_configured_patch() {
     # Get rid of absolute paths in the patch.
     sed -ie '/---\|+++/ { s%${MENDER_UBOOT_OLD_SRC}%a%; s%${S}%b% }' ${WORKDIR}/mender_auto_configured.patch
 
+    # Prepend Patch Upstream Status
```

Comment:
Under which conditions is this necessary? The patches the layer should all have the `Upstream-Status` tag.

Reply:
This is a patch that we auto-generate in this task (see few lines above `diff -r -u ...`) for the user to add it later to his/her recipe. Hence we need to prepend the status.